Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,44 @@
+# Microcontroller Signoff
```

Comment:
"Each subsystem must have at least one constraint arising form standards, ethics, or socioeconomic well being." In which constraint is this addressed?

Reply:
We plan on using USB and SPI/UART as standards in hardware communication.  This is stated in constraint 2 and 3.